examples/thompson_sampling.py

```diff
@@ -63,9 +64,7 @@ def model(X, Y, kernel=matern32_kernel):
 
 
 class GP:
-    """ Adapted to numpyro from https://gdmarmerola.github.io/ts-for-bayesian-optim/ """
```

Comment:
why was this removed?

Reply:
I put this into the ThompsonSamplingGP class, where it makes more sense